packaging/Dockerfile.ubuntu.18.10

```diff
@@ -4,15 +4,21 @@
 # 'recipe' for Docker to build an Debian package
 #
 # Pull base image
-FROM ubuntu:18.04
+FROM ubuntu:18.10
```

Comment:
If we have a build working on 18.04, we should probably leave building.
Ubuntu 18.10 is no longer supported.   Ubuntu 19.04 is current.
See https://github.com/daos-stack/packaging/pull/10

Reply:
I don't see any harm in continuing to build on 18.10 until we have our Ubuntu story figured out.  When we do it will just be an update to _pipeline-lib_, `packaging/` and some `Jenkinsfile`s.  And the latter doesn't even have to be done.  It will just NOOP.